compare_mt/scorers.py

```diff
@@ -437,17 +436,60 @@ def __init__(self, sub_pen=1.0, ins_pen=1.0, del_pen=1.0, case_insensitive=False
     self.del_pen = 1.0
     self.case_insensitive = case_insensitive
 
+  def score_corpus(self, ref, out):
+    """
+    Score a corpus using WER
+
+    Args:
+      ref: A reference corpus
+      out: An output corpus
+
+    Returns:
+      A tuple containing a single value for the WER and None
+    """
+    cached_stats = self.cache_stats(ref, out)
+    return self.score_cached_corpus(np.arange(len(ref)), cached_stats)
+
   def score_sentence(self, ref, out):
+    return self.score_corpus([ref], [out])
+
+  def cache_stats(self, ref, out):
     """
-    Score a single sentence with WER
+    Cache sufficient statistics for caculating WER
 
     Args:
-      ref: A reference sentence
-      out: An output sentence
+      ref: A reference corpus
+      out: An output corpus
 
     Returns:
-      The WER, and None
+      A tuple of cached statistics
     """
+    cached_ref_len = []
```

Comment:
It's probably better to have a list of tuples instead of two separate lists.

Reply:
Thank you! Fixed this!